code/modules/mob/living/basic/ruin_defender/wizard/wizard.dm

```diff
@@ -0,0 +1,104 @@
+/mob/living/basic/wizard
+	name = "Space Wizard"
+	desc = "A wizard is never early. Nor is he late. He arrives exactly at the worst possible moment."
+	icon = 'icons/mob/simple/simple_human.dmi'
+	icon_state = "wizard"
+	icon_living = "wizard"
+	icon_dead = "wizard_dead"
+	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	sentience_type = SENTIENCE_HUMANOID
+	speed = 0
+	maxHealth = 100
+	health = 100
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	attack_verb_continuous = "punches"
+	attack_verb_simple = "punch"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	combat_mode = TRUE
+	habitable_atmos = list("min_oxy" = 5, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
+	unsuitable_atmos_damage = 7.5
+	faction = list(ROLE_WIZARD)
+	basic_mob_flags = DEL_ON_DEATH
+	ai_controller = /datum/ai_controller/basic_controller/wizard
+
+	/// A list of possible wizard corpses, and therefore wizard outfits, to select from
+	var/static/list/wizard_outfits = list(
+		/obj/effect/mob_spawn/corpse/human/wizard,
+		/obj/effect/mob_spawn/corpse/human/wizard/red,
+		/obj/effect/mob_spawn/corpse/human/wizard/yellow,
+		/obj/effect/mob_spawn/corpse/human/wizard/black,
+	)
+	/// A specified wizard corpse spawner to use. If null, picks from the list above instead.
+	var/selected_outfit
+
+	/// The wizard's "main" attack, a targeted spell like fireball
+	var/datum/action/cooldown/spell/targeted_spell
```

Comment:
What are we holding this and `secondary_spell` in memory for? Doesn't look necessary.